.github/workflows/pr_title_check.yaml

```diff
@@ -63,3 +66,16 @@ jobs:
             The subject "{subject}" found in the pull request title "{title}"
             didn't match the configured pattern. Please ensure that the subject
             doesn't start with an uppercase character.
+
+      - name: Check length of PR title
+        env:
+          # Do not use ${{ github.event.pull_request.title }} directly in run command.
+          TITLE: ${{ github.event.pull_request.title }}
+        # We want to make sure that each commit "subject" is under 53 characters not to
+        # be truncated in the git log as well as in the GitHub UI.
+        # https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/process/submitting-patches.rst?id=bc7938deaca7f474918c41a0372a410049bd4e13#n664
+        run: |
+          if (( ${#TITLE} >= 53 )); then
+            echo "The PR title is too long. Please keep it under 52 characters."
```

Comment:
The message doesn’t accurately describe what the check is doing. It should state 53, as “under 52” implies less than 52 to me.